server/cpho/jinja2/indicators/list_indicators.jinja2

```diff
@@ -1,6 +1,8 @@
 {% extends 'base.jinja2' %}
 
 {% block content %}
+    <h2>Hopic Home</h2>
```

Comment:
<h2> should come after <h1>.
If there's no <h1> before it,
Try using <div class='h2'>